### DISCUSSION TITLE
LURO - bad calculation of total win price ( winning price + BONUS) - same number. Total price must be higher than the winning price

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#72] [Bugs and Reports]
<img width="797" alt="Snímek obrazovky 2024-10-21 v 14 25 58" src="https://github.com/user-attachments/assets/4f7588b8-be2c-4bea-b4de-3e24bcf364f7">

**Comment by bf-monster** [comment]
Hello, thank you for report
@bf-plane  check this pls
